src/main/java/net/sknv/qooq/cards/Blackjack.java

```diff
@@ -0,0 +1,26 @@
+package net.sknv.qooq.cards;
+
+import net.dv8tion.jda.api.entities.Message;
+
+public class Blackjack {
+
+    public static boolean bet(Message msg) {
+        String[] text = msg.getContentRaw().split(" ");
```

Comment:
why reparse the params instead of having String[] params passed as a parameter from the parse() method

Reply:
Inicialmente só passava o array mas depois queria ter do outro lado cenas da Message tipo getChannel, getAuthor etc e pronto. Acho q depois com mais módulos/commands podemos dar settle no design de um parser melhor